Sources/SnapshotTesting/Diffable.swift

```diff
@@ -1,14 +1,30 @@
 import Foundation
 
-public struct Diffable<A> {
-  public let to: (A) -> Data
-  public let fro: (Data) -> A
-  public let diff: (A, A) -> (String, [Attachment])?
+/// The ability to compare `Value`s and convert them to and from `Data`.
+public struct Diffable<Value> {
+  /// Converts a value _to_ data.
+  public let to: (Value) -> Data
 
+  /// Produces a value _ from_ data.
+  public let fro: (Data) -> Value
```

Comment:
@mbrandonw Think we should rename any of these? I took `to`/`fro` from a PureScript library because I thought it was cute, but we could expand to `from` or something even more verbose.

Reply:
yeah maybe just `toData` and `fromData`?

`to`/`fro` might be better names if we were to use an `Iso<A, B>` type.